content/documentation/staging/features/index.adoc

```diff
@@ -796,6 +796,26 @@ Kiali has different validations to help troubleshoot configurations related to `
 </div>
 ++++
 
+== Istio Component Status
+
+The Istio service mesh architecture is based in different components: from the istiod to Jaeger. Those components are key for the mesh to work well. That is why Kiali runs regular checks looking for the status of these Istio components.
```

Comment:
Please replace with:
The Istio service mesh architecture is comprised of several components, from istiod to Jaeger.  Each component must work as expected for the mesh to work well overall. Kiali regularly checks the status of each Istio component to ensure the mesh is healthy.